src/calculateRentalCost.js

```diff
@@ -13,9 +13,9 @@ function calculateRentalCost(days) {
   const basePrice = days * PRICE_PER_DAY;
 
   if (days >= LONG_TERM) {
-    return basePrice - LONG_TERM_DISCOUNT;
+   return basePrice - LONG_TERM_DISCOUNT;
   } else if (days >= SHORT_TERM) {
```

Comment:
[Check list]: DON'T add `else` after `if` with the `return` - the code after it won't be executed anyway.

BAD EXAMPLE:

if (condition) {
  return x;
} else {
  return y;
}

GOOD EXAMPLE:

if (condition) {
  return x;
}

return y;